autogpt/commands/file_operations.py

```diff
@@ -321,3 +323,15 @@ def list_files(directory: str, agent: Agent) -> list[str]:
             found_files.append(relative_path)
 
     return found_files
+
+
+def file_search_args(input_args: dict[str, any], agent: Agent):
+    # Force only searching in the workspace root
+    input_args["dir_path"] = str(agent.workspace.root)
```

Comment:
Can we make this more flexible? e.g. `input_args["dir_path"] = str(agent.workspace.get_path(input_args["dir_path"]))`